backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQExtension.java

```diff
@@ -71,24 +75,27 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
         return rabbitMQ;
     }
 
-    public RabbitChannelPoolImpl getRabbitChannelPool() {
+    public RabbitChannelPoolImpl getRabbitChannelPoolImpl() {
```

Comment:
I find it weird to have a `getRabbitChannelPoolImpl` and a `getSimpleChannelPool`

I would propose those solutions:
 - 1 Always pass the simple channel pool as it is known as safe
 - Or 2 Return either one of the other depending on an enum passed to the constructor.

Anyway we should have a single `RabbitMQChannelPool getChannelPool()` method IMO

(and I do prefer option 1...)

Reply:
The reason I did keep `rabbitChannelPoolImpl` instance in my extension, beacause I saw on master branch, there are some test still use it. Now I will replace completely by `simpleChannelPool`